### ISSUE TITLE
updating a game should completely wipe out the previous install of it

### ISSUE THREAD

**Issue by brine** [octgn/OCTGN#914]
Now that the card images are separated from the game's install directory, we need the game database to be completely wiped clean each time a game gets an update (via the feed, usually).

For example, if I were to delete a python file, or change the background image, or change a set's GUID, then after OCTGN updates the game the old files/folders will still exist in the game directory.  This shouldn't be happening, especially in the case of changing the set's GUID as you'd now essentially have a duplicate set in the database.

So tl;dr: updating a game via the feed should completely wipe the game from the database, and reinstall the new one.

**Comment by brine** [comment]
The issue with the /Decks folders will be fixed by #1764
